maths/numerical_analysis/adams_bashforth.py

```diff
@@ -0,0 +1,199 @@
+"""
+Use the Adams-Bashforth methods to solve Ordinary Differential Equations.
+
+
+https://en.wikipedia.org/wiki/Linear_multistep_method
+Author : Ravi Kumar
+"""
+from collections.abc import Callable
+
+import numpy as np
+
+
+class AdamsBashforth:
+    def __init__(
+        self,
+        func: Callable[[float, float], float],
+        x_initials: list[float],
+        y_initials: list[float],
+        step_size: float,
+        x_final: float,
+    ):
```

Comment:
```suggestion
    ) -> None:
```
Or should this be a https://docs.python.org/3/library/dataclasses.html instead?